apps/agora/server/project.json

```diff
@@ -0,0 +1,97 @@
+{
+  "name": "agora-server",
+  "$schema": "../../../node_modules/nx/schemas/project-schema.json",
+  "sourceRoot": "apps/agora/server/src",
+  "projectType": "application",
+  "targets": {
+    "create-config": {
+      "executor": "nx:run-commands",
+      "options": {
+        "command": "cp -n .env.example .env",
+        "cwd": "{projectRoot}"
+      }
+    },
+    "build": {
+      "executor": "@nx/webpack:webpack",
+      "outputs": ["{options.outputPath}"],
+      "defaultConfiguration": "production",
+      "options": {
+        "target": "node",
+        "compiler": "tsc",
+        "outputPath": "dist/apps/agora/server",
+        "main": "apps/agora/server/src/main.ts",
+        "tsConfig": "apps/agora/server/tsconfig.app.json",
+        "assets": ["apps/agora/server/src/assets"],
+        "webpackConfig": "apps/agora/server/webpack.config.js",
+        "generatePackageJson": true
```

Comment:
`generatePackageJson` will include a `package.json` in the build, which can be used to install the necessary dependencies in the Docker image